ngraph/test/models/onnx/group_norm.prototxt

```diff
@@ -0,0 +1,108 @@
+ir_version: 3
+producer_name: "nGraph ONNX Importer"
+graph {
+  node {
+    input: "data"
+    input: "gamma"
+    input: "beta"
+    output: "y"
+    op_type: "GroupNorm"
+    domain: "org.openvinotoolkit"
+    attribute {
+        name: "num_groups"
+        i: 4
+        type: INT
+    }
+    attribute {
+        name: "eps"
+        f: 1e-6
+        type: FLOAT
+    }
+  }
+  name: "group_norm_example"
+  initializer {
+    dims: 8
+    data_type: 1
+    name: "gamma"
+    raw_data: "\0\0\200?\0\0\0@\0\0@@\0\0\200@\0\0\240@\0\0\300@\0\0\340@\0\0\0A"
+  }
+  initializer {
+    dims: 8
+    data_type: 1
+    name: "beta"
+    raw_data: "\0\0\200?\0\0\0@\0\0@@\0\0\200@\0\0\240@\0\0\300@\0\0\340@\0\0\0A"
+  }
+  input {
+    name: "data"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: 2
+          }
+          dim {
+            dim_value: 8
+          }
+          dim {
+            dim_value: 2
+          }
+          dim {
+            dim_value: 2
+          }
+        }
+      }
+    }
+  }
+  input {
+    name: "gamma"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: 8
+          }
+        }
+      }
+    }
+  }
+  input {
+    name: "beta"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: 8
+          }
+        }
+      }
+    }
+  }
+  output {
+    name: "y"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: 2
+          }
+          dim {
+            dim_value: 8
+          }
+          dim {
+            dim_value: 2
+          }
+          dim {
+            dim_value: 2
+          }
+        }
+      }
+    }
+  }
+}
+opset_import {
+  version: 1
+}
```

Comment:
Can we pass this file to MO and compare results from MO and ONNX importer for it? I wasn't able to pass this file to MO, text pb is not supported. @mateusztabaka, could you suggest a tool to make it possible?

Reply:
you can use this tool https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/models/onnx/onnx_prototxt_converter.py

Reply:
@evgenytalanin-intel, please consider using this tool to sync coverage between the importer and MO.

Reply:
@mateusztabaka, thanks, works perfectly for me. But still there is a question about coverage and comparison to MO. Can @evgenytalanin-intel help us?